plugin/documents.py

```diff
@@ -657,6 +664,62 @@ def render_highlights_on_main_thread() -> None:
 
         sublime.set_timeout(render_highlights_on_main_thread)
 
+    # --- textDocument/semanticTokens -------------------------------------------------------------------------------
+
+    def _do_semantic_tokens_async(self) -> None:
+        if not userprefs().enable_semantic_tokens:
+            return
+        if "background" not in self.view.style_for_scope("meta.semantic-token.lsp"):
+            color_scheme_error()
+            return
```

Comment:
I think that we shouldn't be triggering this error/warning every time the tokens are requested.
Maybe we should only have this information in the documentation and the documentation should be linked from the description of the setting that enables this feature.